### PR TITLE
Fixes to SVG generation

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: client/card-images/createAllCards.ts [hunk 1]
```diff
@@ -11,17 +11,17 @@ import drawCards from "../src/game/ui/drawCards";
 import * as drawCardsNode from "./drawCardsNode";
 
 // Get the "No Variant" variant
```
Comment (Zamiell) [inline client/card-images/createAllCards.ts:13]: this comment is wrong

File: client/card-images/createAllCards.ts [hunk 2]
```diff
@@ -56,18 +56,14 @@ for (const [key, value] of allCardImages.entries()) {
...
-    }
-
     if (type === "card") {
       if (suit === "Unknown") {
         // We only care about the real unknown ranks
         if (rank >= 1 && rank <= 5) {
           fileName = rank.toString();
         }
       } else {
+	let suitAbbrev = variant.suits.find(s => s.name === suit)?.abbreviation.toLowerCase();
```
Comment (Zamiell) [inline client/card-images/createAllCards.ts:66]: is this indented properly?

**Comment by DarthGandalf** [comment]
Fixed.

**Comment by DarthGandalf** [comment]
No, I haven't had a chance to run it locally yet

**Comment by Zamiell** [comment]
looks tested now, thanks

**Pull request merged**